eng/sendtohelix.proj

```diff
@@ -16,6 +16,7 @@
     <!-- We need to enable xunit reporter so that it parses test results
          Package testing doesn't run on xunit. -->
     <EnableXunitReporter Condition="'$(TargetGroup)' != 'AllConfigurations'">true</EnableXunitReporter>
+    <EnableAzurePipelinesReporter Condition="'$(TargetGroup)' != 'AllConfigurations'">true</EnableAzurePipelinesReporter>
```

Comment:
This is passed as a global property on helix.yml step so it would be overridden. You need to remove it from there.

Reply:
That is why you don’t see any test results in this PR build.

Also we don’t want to use the azure test explorer for official builds since we don’t wait for the test execution to finish and that causes trouble with the test explorer. In order to enable the test explorer in official builds we need to split the build/publish into one stage and then run the tests afterwards into another stage where build/publishing is not dependent of the test results but the build DO wait for tests to finish running.